cmd/main.go

```diff
@@ -0,0 +1,121 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Cloud Container Service, 5737-D43
+ * (C) Copyright IBM Corp. 2019, 2020 All Rights Reserved.
+ * The source code for this program is not  published or otherwise divested of
+ * its trade secrets, irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+
+//Package main ...
+package main
+
+import (
+	"flag"
+	libMetrics "github.com/IBM/ibmcloud-volume-interface/lib/metrics"
+	csiConfig "github.com/IBM/satellite-object-storage-plugin/config"
+	driver "github.com/IBM/satellite-object-storage-plugin/pkg/driver"
+	"github.ibm.com/alchemy-containers/ibm-csi-common/pkg/utils"
+	//"github.com/ctrox/csi-s3/pkg/s3"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.ibm.com/alchemy-containers/ibm-csi-common/pkg/metrics"
```

Comment:
The `ibm-csi-common` is from IBM internal private repository, the pkg should not be used here.